boxsdk/client/client.py

```diff
@@ -3,24 +3,34 @@
 from __future__ import unicode_literals
 import json
 
-from .auth import DeveloperTokenAuth
-from .config import API
-from .session.box_session import BoxSession
-from .network.default_network import DefaultNetwork
-from .object.user import User
-from .object.folder import Folder
-from .object.search import Search
-from .object.events import Events
-from .object.file import File
-from .object.group import Group
-from .object.group_membership import GroupMembership
-from .util.shared_link import get_shared_link_header
-from .util.translator import Translator
+from ..auth import OAuth2
+from ..config import API
+from ..session.box_session import BoxSession
+from ..network.default_network import DefaultNetwork
+from ..object.user import User
+from ..object.folder import Folder
+from ..object.search import Search
+from ..object.events import Events
+from ..object.file import File
+from ..object.group import Group
+from ..object.group_membership import GroupMembership
+from ..util.shared_link import get_shared_link_header
+from ..util.translator import Translator
+from ..util.url import get_url
 
 
 class Client(object):
 
-    def __init__(self, oauth, network_layer=None, session=None):
```

Comment:
I'd prefer to leave the oauth param required, than have ``Client.__init__`` slowly gain all of the params from ``OAuth.__init__``. And if you already have these params available, it really is barely more convenient to pass them straight to `Client`, as opposed to passing them to `OAuth2` which is passed to `Client`. I think we lose a lot more cluttering up the param list, than we gain in convenience.